docs/getting_oss_rules_of_play_rating.md

```diff
@@ -0,0 +1,170 @@
+that would positively impact the rating.
+
+If `--interactive` option is specified, the tool becomes a bit interactive,
+and may ask the user a couple of questions about the project.
+
+### Creating GitHub issues for failed rules
+
+Exclusively for the OSS rules of play rating, Fosstars can create GitHub issues for failed rules directly in the affected repositories so the issues can be followed up by the repository owners. If you want to do that, add the command-line option `--create-issues`. Fosstars will inform you on the command-line about the issue creation process as well:
+
+```
+[+] Creating issues for findings on https://github.com/<organization>/<repository>
+[+] New issue: [rop-license_file-3] Violation against OSS Rules of Play
+[+] New issue: [rop-vulnerability_alerts-1] Violation against OSS Rules of Play
+[+] New issue: [rop-reuse_tool-2] Violation against OSS Rules of Play
+[+] New issue: [rop-security_policy-1] Violation against OSS Rules of Play
+[+] New issue: [rop-license_file-2] Violation against OSS Rules of Play
+[+] New issue: [rop-reuse_tool-1] Violation against OSS Rules of Play
+[+] New issue: [rop-reuse_tool-3] Violation against OSS Rules of Play
+[+] New issue: [rop-reuse_tool-4] Violation against OSS Rules of Play
+```
+
+Of course, it won't create duplicates of already existing GitHub issues. Fosstars uses the previously defined rule IDs to check if the repository already contains an issue with the same ID that is not closed yet. In such cases, the command-line output will also report that accordingly:
+
+```
+[+] Creating issues for findings on https://github.com/<organization>/<repository>
+[+] Issue already existing: [rop-license_file-3] Violation against OSS Rules of Play
+[+] New issue: [rop-vulnerability_alerts-1] Violation against OSS Rules of Play
+[+] Issue already existing: [rop-reuse_tool-2] Violation against OSS Rules of Play
+[+] Issue already existing: [rop-security_policy-1] Violation against OSS Rules of Play
+[+] Issue already existing: [rop-license_file-2] Violation against OSS Rules of Play
+[+] Issue already existing: [rop-reuse_tool-1] Violation against OSS Rules of Play
+[+] Issue already existing: [rop-reuse_tool-3] Violation against OSS Rules of Play
+[+] Issue already existing: [rop-reuse_tool-4] Violation against OSS Rules of Play  
+```
+
+### Other options
+
+Other command-line options are as well available for use with the OSS rules of play rating. Please refer to the [security rating documentation](getting_oss_security_rating.md) for more details.
+
+## Fosstars OSS Rules of Play GitHub action
+
+SAP already uses the OSS Rules of Play rating in a [GitHub action](https://github.com/SAP/fosstars-rating-core/blob/master/.github/workflows/fosstars-rules-of-play.yml) and posts the results in a dedicated branch on [fosstars-rating-core](https://github.com/SAP/fosstars-rating-core/tree/fosstars-oss-rules-of-play-sap-report).
```

Comment:
[`fosstars-rating-core`](https://github.com/SAP/fosstars-rating-core/tree/fosstars-oss-rules-of-play-sap-report)
- Lets have a common naming convention. In the top most section you use `fosstars-oss-rules-of-play-sap-report`
- Also lets have normal mention instead of an identifier syntax. For example: **Rules of Play report from Fosstars**
- I leave it up to you as how you see fit.
- It would be really nice if the point 2 is applied everywhere.

Reply:
The first identifier is simply the branch for the SAP report. From my perspective, the more relevant identifier here would be the command-line option `oss-rules-of-play` - that's how Artem called the rating at the first place. That's why I explained everything in the documentation accordingly. 
I'm not against a consistent renaming, but from my perspective also the focus on _open source_ rules of play makes slightly more sense than mentioning that the report comes from Fosstars. Thus, I'd rather keep it as it is right now.